dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -154,7 +154,7 @@ def model_install_cmds(output_file=None) -> str:
           f"pip3 install --user --pre {version_mapping.TORCH.value} {version_mapping.TORCHVISION.value} {version_mapping.TORCHAUDIO.value} --index-url {version_mapping.TORCH_INDEX_CPU_URL.value}"
       ),
       (
-          f"pip3 install --user 'torch_xla[tpuvm] @{version_mapping.TORCH_XLA_TPU_WHEEL.value}'"
+          f"pip3 install --user 'torch_xla[tpu] @{version_mapping.TORCH_XLA_TPU_WHEEL.value}' -f https://storage.googleapis.com/libtpu-releases/index.html"
```

Comment:
Will this also automatically install `torch_xla` if `torch_xla` is not installed?

Reply:
Yeah

Reply:
Nice, LGTM!